.github/workflows/materialized-views.yml

```diff
@@ -0,0 +1,54 @@
+---
+name: Materialized views create
+on:
+  schedule:
+    # Invoke at 6 UTC EST every day
+    - cron: '0 6 * * *'
+  workflow_dispatch:
+    inputs:
+      environment:
+        required: true
+        type: choice
+        description: The environment the workflow should run on.
+        options:
+          - dev
+          - staging
+          - preview
+          - production
+
+jobs:
+  scheduled-materialized-views:
+    if: ${{ github.event_name == 'schedule' }}
```

Comment:
Went with a check against the event context here to avoid duplicating the cron expression.